src/web-ui/src/public/Checkout.vue

```diff
@@ -15,151 +15,88 @@
+                    <span>Total before tax:</span>
+                    <strong>${{ this.cartTotalBeforeTax.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between border-top-0">
+                    <span>Tax to be collected:</span>
+                    <strong>${{ this.cartTaxRate.toFixed(2) }}</strong>
+                  </li>
+                  <li class="list-group-item p-1 d-flex justify-content-between border-bottom-0">
+                    <span>Order total:</span>
+                    <strong>${{ this.cartTotal.toFixed(2) }}</strong>
+                  </li>
+                </ul>
+                <button class="btn btn-primary btn-lg btn-block" v-on:click="submitOrder">Place your Order</button>
               </div>
-            </form>
+            </div>
+
+            <button v-if="pinpointEnabled && user" v-on:click="triggerAbandonedCartEmail" class="abandoned-cart-btn btn btn-primary btn-lg m-4">
+              Trigger Abandoned Cart email
+            </button>
           </div>
           <div class="col-md-8 order-md-1">
-            <h4 class="d-flex justify-content-between align-items-center mb-3">
-              <span class="text-muted">Details</span>
-            </h4>          
-            <hr/>
-            <h5 class="mb-3">Billing Address</h5>
-            <form>
-              <div class="row">
-                <div class="col-md-6 mb-3">
-                  <label for="firstName">First name</label>
-                  <input type="text" class="form-control" id="firstName" v-model="order.billing_address.first_name" placeholder="" value="" required>
-                </div>
-                <div class="col-md-6 mb-3">
-                  <label for="lastName">Last name</label>
-                  <input type="text" class="form-control" id="lastName" v-model="order.billing_address.last_name" placeholder="" value="" required>
-                </div>
-              </div>
-
-              <div class="mb-3">
-                <label for="email">Email <span class="text-muted">(Optional)</span></label>
-                <input type="email" class="form-control" id="email" v-model="order.email" placeholder="you@example.com">
-              </div>
-
-              <div class="mb-3">
-                <label for="address">Address</label>
-                <input type="text" class="form-control" id="address" v-model="order.billing_address.address1" placeholder="1234 Main St" required>
-              </div>
-
-              <div class="mb-3">
-                <label for="address2">Address 2 <span class="text-muted">(Optional)</span></label>
-                <input type="text" class="form-control" id="address2" v-model="order.billing_address.address2" placeholder="Apartment or suite">
-              </div>
+            <div class="alert alert-secondary text-center ml-0" v-if="showCheckout == true">This storefront is not real.<br/>Your order will not be fulfilled.</div>
```

Comment:
This div should be light blue. Let's add `blue-100: #f1faff;` to the tokens css file and use that as the background